test/src/test/resources/fixtures/new hydration/new batching/new-batching-conditional-hydration-accepts-true-condition-after-a-false-condition.yml

```diff
@@ -0,0 +1,150 @@
+name: "new batching conditional hydration accepts true condition after a false condition"
+enabled: true
+overallSchema:
+  service2: |
+    type Query {
+      barsById(ids: [ID]): [Bar]
+    }
+    type Bar {
+      id: ID
+      name: String
+    }
+  service1: |
+    type Query {
+      foo: Foo
+    }
+    type Foo {
+      id: ID
+      type: String
+      bar: [Bars] 
+        @hydrated(
+          service: "service2"
+          field: "barsById"
+          arguments: [
+            {
+              name: "ids"
+              value: "$source.barIds"
+            }
+          ]
+          when: {
+            result: {
+              sourceField: "type"
+              predicate: { equals: "thatType" }
+            }
+          }
+        )
+        @hydrated(
+          service: "service2"
+          field: "barsById"
```

Comment:
How do we tell which instruction is being used if the `@hydrated` instructions are using the same info?

i.e. I think the test will still pass even if the other `@hydrated` instruction is selected.

Reply:
the two different hydrations are now hydrating from a different field 👍